server/src/main/java/fr/uga/l3miage/tp4/models/ExamEntity.java

```diff
@@ -0,0 +1,34 @@
+package fr.uga.l3miage.tp4.models;
+
+import fr.uga.l3miage.tp4.enums.SessionStatus;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+public class ExamEntity {
+
+    @Id
+    @Column(name="id")
+    private Long id;
+
+    @Column(unique=true)
+    private String name;
+
+    private LocalDateTime startDate;
+
+    private LocalDateTime endDate;
+
+    private SessionStatus status;
+
+    @OneToMany(mappedBy="exam")
+    private Set<CandidateEvaluationGridEntity> candidateEvaluationGrids;
+
+    @ManyToMany(mappedBy = "exams")
+    private Set<SkillEntity> skills;
```

Comment:
je la voulais aussi en uni
```suggestion
    @ManyToMany
    private Set<SkillEntity> skills;
```